.jenkins/Jenkinsfile

```diff
@@ -60,15 +60,15 @@ def ACCContainerTest(String label, String version) {
                 // Run the OE tests from the git repository with the currently
                 // generated az-dcap-client deb package installed
                 def task = """
-				
+                
                         sudo apt-get update -y
                         sudo apt-get install -y libgtest-dev
                         sudo apt-get install -y cmake
                         cd /usr/src/gtest
                         sudo cmake CMakeLists.txt
                         sudo make
-                        sudo cp *.a /usr/lib	
-						
+                        sudo cp *.a /usr/lib 2>/dev/null|| sudo cp lib/*.a /usr/lib
```

Comment:
Ignoring failures is not a great practice, and your 2nd command can still fail.

An example to do it in a more robust way:
```
find . ./lib -name "*.a" -exec cp -t /usr/lib {} +
```

Repeat for all occurrences